src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -12,12 +12,114 @@
 #include "include/buffer.h"
 #include "rbm_device.h"
 #include "nvme_block_device.h"
+#include "block_rb_manager.h"
 
 namespace {
   seastar::logger& logger() {
     return crimson::get_logger(ceph_subsys_seastore_tm);
   }
 }
+namespace crimson::os::seastore::random_block_device {
+#include "crimson/os/seastore/logging.h"
+SET_SUBSYS(seastore_device);
+
+RBMDevice::mkfs_ret RBMDevice::mkfs(device_config_t config) {
+  LOG_PREFIX(RBMDevice::mkfs);
+  super.start = 0;
```

Comment:
I renamed RBM_START_ADDRESS. CBJournal has a fixed start position (at the beginning of the device). On the other hand, RBM has a variable position because the two modules coexist on a single device. So, we need super.start where the RBM begins.

Reply:
Regarding to https://github.com/ceph/ceph/pull/48478#discussion_r995621995, if the journal area always starts at the block-size, probably we only need to remember the journal-size in the device's superblock, replacing the two existing superblock members `rbm_metadata_header_t::start/start_data_area`, they can be inferred instead.

CBJournal can load the journal range from the backing RBDevice after device mkfs/mount.

Reply:
Done.